app/workers/evss/disability_compensation_form/submit_uploads.rb

```diff
@@ -17,24 +17,36 @@ def self.start(uuid)
         batch.jobs do
           claim_id = get_claim_id(uuid)
           uploads = get_uploads(uuid)
-          uploads.each { |u| perform_async(uuid, u[:guid], claim_id) }
+          uploads.each_with_index do |upload_data, _index|
+            perform_async(upload_data, claim_id, uuid)
+          end
         end
       end
 
-      def self.get_claim_id(_uuid)
-        nil
-      end
-
-      def self.get_uploads(_uuid)
-        nil
+      def self.perform(upload_data, claim_id, uuid)
+        user = User.find(uuid)
+        auth_headers = EVSS::AuthHeaders.new(user).to_h
+        client = EVSS::DocumentsService.new(auth_headers)
+        file_body = AncillaryFormAttachment.find_by(guid: upload_data[:guid]).file_data
+        document_data = EVSSClaimDocument.new(
+          evss_claim_id: claim_id,
+          file_name: upload_data[:file_name],
+          tracked_item_id: nil,
+          document_type: upload_data[:doctype]
+        )
+        client.upload(file_body, document_data)
+      rescue StandardError => error
+        raise error
       end
 
-      def perform(uuid, guid, claim_id)
-        # TODO: process upload
+      def self.get_claim_id(uuid)
+        form_submission = ::DisabilityCompensationSubmission.find_by(user_uuid: uuid, form_type: FORM_TYPE)
+        form_submission.claim_id
       end
 
-      def on_success(_status, _options)
-        # TODO: send email notification
+      def self.get_uploads(uuid)
+        user = User.find(uuid)
```

Comment:
we get the user object in both `get_uploads` and `perform` we could get it once in `start` and pass it rather than `uuid` (`uuid` is used in callbacks because they only support shallow hashes)